### PR TITLE
Updating paper to new structure.

### PR DESCRIPTION
**Pull request by arfon**

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -9,12 +9,16 @@ tags:
 authors:
  - name: Christos Theodoropoulos
    orcid: 0000-0002-5395-0347
-   affiliation 1: Hellenic Centre for Marine Research
-   affiliation 2: National Technical University of Athens
+   affiliation: 1, 2
  - name: Nikolaos Skoulikidis
```
Comment (arfon) [inline paper/paper.md:13]: @chtheodoro - could you also add ORCIDs for these two authors please?

Comment (chtheodoro) [reply paper/paper.md:13]: I added the ORCID of Anastasios Stamou, however, the other author does not have one. If it is necessary, please let me know ok?

**Review by chtheodoro — APPROVED** [review]

**Review by chtheodoro — APPROVED** [review]

File: paper/paper.md (hunk 1 above)
Comment (arfon) [reply paper/paper.md:13]: We strongly prefer that authors add ORCIDs if they have an academic affiliation.

Comment (chtheodoro) [reply paper/paper.md:13]: Ok I will make sure he sings up for one and fill it in the paper.md

Comment (chtheodoro) [reply paper/paper.md:13]: @arfon all orcid id numbers are added.

**Review by chtheodoro — APPROVED** [review]

File: paper/paper.md (hunk 1 above)
Comment (arfon) [reply paper/paper.md:13]: 👍 looks great

Comment (chtheodoro) [reply paper/paper.md:13]: @arfon I can't merge the branch to apply the changes... Am I doing anything wrong?

**Pull request merged**